File: solutions/0824-number-of-lines-to-write-string/solution.py
class Solution(object):
    def numberOfLines(self, width, s):
        """
        :type widths: List[int]
        :type s: str
        :rtype: List[int]
        """
        line=0
        p=0
        k=0;
        for a in s:
            if(p+width[ord(a)-97]>100):
                line+=1;
                p=0;
            p+=width[ord(a)-97];
        return [line+1,p]
        

File: solutions/0824-number-of-lines-to-write-string/solution.java
class Solution {
    public int[] numberOfLines(int[] width, String s) {
        int line=0,p=0,k=0;
        for(char a:s.toCharArray())
        {
            if(p+width[a-97]>100)
            {
                line++;
                p=0;
            }
            p+=width[a-97];
        }
        return new int[] {line+1,p};
    }
}
